docs/sources/articles/networking.md

```diff
@@ -105,7 +105,7 @@ Finally, several networking options can only be provided when calling
     [Configuring DNS](#dns) and
     [How Docker networks a container](#container-networking)
 
- *  `--link=CONTAINER_NAME:ALIAS` — see
+ *  `--link=<CONTAINER_NAME|CONTAINER_ID>:ALIAS` — see
```

Comment:
Perhaps this should be `CONTAINER_NAME_or_ID` to be (more) consistent with `--net` below on line 112?